### PR TITLE
Changed the fractional digits to display 6 places

### PR DESCRIPTION
**Pull request by nd4p90x**
Issue #223 Update the default originalTimestamp to send 6 decimal places to the server

### CODE REVIEW

File: lib/segment/analytics/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Segment
   class Analytics
-    VERSION = '2.2.8.pre'
```
Comment (pooyaj) [inline lib/segment/analytics/version.rb:3]: @nd4p90x Do you think we need a major version bump here, given we dropped support for older versions of Ruby?

**Review by pooyaj — COMMENTED** [review]
awesome, looks great 💥 let's decide on the versioning and then merge

File: lib/segment/analytics/version.rb (hunk 1 above)
Comment (nd4p90x) [reply lib/segment/analytics/version.rb:3]: I was updating the version file to match the actual history, we moved out of -pre in December 2020

Comment (nd4p90x) [reply lib/segment/analytics/version.rb:3]: I update the minor version with history

**Review by pooyaj — APPROVED** [review]

**Pull request merged**